src/m365/teams/commands/channel/channel-remove.ts

```diff
@@ -140,14 +130,52 @@ class TeamsChannelRemoveCommand extends GraphCommand {
         type: 'confirm',
         name: 'continue',
         default: false,
-        message: `Are you sure you want to remove the channel ${channelName} from team ${args.options.teamId}?`
+        message: `Are you sure you want to remove the channel ${channelName}?`
       });
-      
+
       if (result.continue) {
         await removeChannel();
       }
     }
   }
+
+  private async getTeamId(args: CommandArgs): Promise<string> {
+    if (args.options.teamId) {
+      return args.options.teamId;
+    }
+
+    const group: Group = await aadGroup.getGroupByDisplayName(args.options.teamName!);
+
+    if ((group as ExtendedGroup).resourceProvisioningOptions.indexOf('Team') === -1) {
+      throw 'The specified team does not exist in the Microsoft Teams';
+    }
+    else {
+      return group.id!;
+    }
+  }
+
+  private async getChannelId(args: CommandArgs): Promise<string> {
+    if (args.options.id) {
+      return args.options.id;
+    }
+
+    const channelRequestOptions: any = {
+      url: `${this.resource}/v1.0/teams/${encodeURIComponent(this.teamId)}/channels?$filter=displayName eq '${encodeURIComponent(args.options.name as string)}'`,
```

Comment:
Instead of `args.options.name as string` you can write it like this `args.options.name!`. This will help typescript know you assume that this value is filled here.